src/FakeItEasy.Analyzer/Directory.Build.props

```diff
@@ -9,10 +9,7 @@
 
   <ItemGroup>
     <!-- Analyzer installation scripts -->
-    <Content Include="tools/*.ps1">
-      <Pack>true</Pack>
-      <PackagePath>tools/%(Filename)%(Extension)</PackagePath>
-    </Content>
+    <Content Include="tools/*.ps1" Pack="true" PackagePath="tools/%(Filename)%(Extension)" />

```

Comment:
This doesn't actually change anything; I just changed the elements to attributes, because it's less verbose.